firewall/handlers/blacklist_handler.go

```diff
@@ -0,0 +1,58 @@
+package handlers
+
+import (
+	"errors"
+
+)
+
+type BlacklistHandler struct {
+	blackQueries[] string
+}
+
+var ErrQueryInBlacklist = errors.New("query in blacklist")
+
+func NewBlacklistHandler(blackQueries []string) (*BlacklistHandler, error) {
+
+	return &BlacklistHandler{blackQueries:blackQueries}, nil
+}
+
+func(handler * BlacklistHandler) CheckQuery(query string) error {
+	yes, _ := handler.contains(query)
+	if yes {
+		return ErrQueryInBlacklist
+	}
+	return nil
+}
+
+func(handler * BlacklistHandler) AddQueriesToBlacklist(queries []string) {
+
+	for index := 0; index < len(queries); index++ {
```

Comment:
```
for _, query := range queries{
  handler.blackQueries = append(handler.blackQueries, query)
}
```
looks shorter and more readable, did it?

Reply:
Agree. Changed